tools/nnpackage_tool/qnf/requirements.txt

```diff
@@ -0,0 +1,3 @@
+numpy>=1.18.5
+circle-schema>=0.4.0.dev0
```

Comment:
It uses `circle-schema`, which is not available in public yet. But it is just a `flatc` generated python code. Someone who is familiar with `flatbuffers` may generate this code using `circle_schema.fbs`. Or someone may use the internal package.